package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "4.6.0",
+  "version": "4.6.1",
```

Comment:
Karim has a PR with the same version change (and updated dependencies below)... I expect he will merge before me, so I'll update this file accordingly.